content/4. Deeltaken/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Boolean/1. Boolean.md

```diff
@@ -0,0 +1,120 @@
+---
+title:
+  - 1. Boolean
+taxonomie:
+  - ib-19.2.Datatype-Primitief-Boolean.DT
+  - ib-19.3.Datatype-Primitief-Boolean.DT
+draft: false
+---
+
+> Meer informatie over [[4. Deeltaken/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Boolean/1. Boolean|Boolean]]
+
+## Opdracht 1
+Maak een boolean aan en print het resultaat.
```

Comment:
Suggestie: 'een variabele van het type 'boolean''. Om aan te duiden dat het nog steeds om een variabele gaat.
Opmerking: zijn dingen als printen naar console al ergens aan bod gekomen? Indien nodig: verwijzing?

Reply:
Dat is inderdaad een betere verwoording. Zover ik weet wordt er verder nog niks gedaan qua printen naar de console. Maar mogelijk nog een goede om aan toe te voegen.